lighthouse-core/audits/metrics/largest-contentful-paint.js

```diff
@@ -44,6 +44,27 @@ class LargestContentfulPaint extends Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts.TraceNode[]} traceNodes
+   * @return {LH.Audit.Details.Table['items']}
+   */
+  static getNodeData(traceNodes) {
+    const lcpNode = traceNodes.find(node => node.type === 'lcp');
```

Comment:
seeing this `type` so close to the `type` thats on L59 makes me think this one should have a different name..  we could go all the way to `auditRef` or stay with a fairly loose `tag`.

Reply:
Opting for `metricTag` as a replacement

Reply:
Not sure where `tag` came from, but I don't like it. It's being used as an `id`, but we could just be specific and say `metric`, since we only plan to have trace nodes that associate with exactly one metric. If that does not hold true sometime in the future, that's fine and we can just change it to `id` with no issue since this is just an artifact.